docs/book/src/cronjob-tutorial/testdata/emptymain.go

```diff
@@ -97,3 +98,26 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+/*
+
+Note that the Manager can restrict the namespace that all controllers will watch for resources by:
+
+```go
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	Scheme: scheme,
+	Namespace: namespace,
+	MetricsBindAddress: metricsAddr}
+)
+```
+
+The above example will change the scope of your project to a Namespaced one and then, in this scenario
+would also be suggested replace the default ClusterRole and ClusterRoleBinding to Role and RoleBinding respectively.
+For further information see the kubernetes documentation about [Using RBAC Authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+
+<aside class="note">
+<h1>Watching a list of Namespaces</h1>
+See [here](https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/cache#MultiNamespacedCacheBuilder) that for some scenarios is possible to watch a list of namespaces as well.
```

Comment:
```
Watching a list of namespaces is possible for some scenarios as explained
[here](https://www.mdpi.com/journal/sensors/special_issues/cloud_computing_iot).
```

Reply:
We probably should make it more clear how to do it. e.g.
```go
mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
	Scheme: scheme,
	NewCache: cache.MultiNamespacedCacheBuilder(myNamespaceList),
	MetricsBindAddress: metricsAddr,
})
```